pkg/service/healthcheck/config.go

```diff
@@ -18,19 +18,19 @@ var (
 
```

Comment:
https://github.com/scylladb/scylla-manager/blob/1ab2ff6f2f060282c73556e7bb2340820610a987/pkg/service/healthcheck/config.go#L10-L17

Default values specified in config are not used anymore - it changed a long time ago, but maybe we should finally remove them?